source/reference/mongotranslate.txt

```diff
@@ -0,0 +1,125 @@
+==================
+``mongotranslate``
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. only:: html
+
+   .. meta::
+      :description: The mongotranslate reference page.
+      :keywords: mongotranslate, mongodb, man page
+
+Description
+-----------
+
+.. only:: (not man)
+
+   .. binary:: mongotranslate
+
+``mongotranslate`` is a learning and troubleshooting tool designed to
+help users understand how |bi-short| translates SQL queries for use
```

Comment:
Hm, I still think this misrepresents the tool's purpose. The part I don't like is "helps users understand how the BI Connector translates SQL Queries". Its primary purpose is not to help people understand how the BI Connector works. Its purpose is to help users learn the MongoDB aggregation language, and it does this by leveraging BIC's translation logic.

Reply:
Perhaps something along these lines?

`mongotranslate` is a learning tool designed to help users understand how SQL queries can be expressed in the MongoDB aggregation language. The translations are provided by the BI Connector's SQL-to-aggregation translation engine, and can also be used to troubleshoot specific BI Connector translation issues.

Reply:
cc: @dmhilly
Thanks @rychipman, I've updated the introductory paragraph.